core/storage/suggestion/gae_models.py

```diff
@@ -661,21 +653,23 @@ def get(cls, strict=True):
             ReviewerAndSuggestionCountsModel.
         """
         reviewer_and_suggestion_counts_model = cls.get_by_id(
-            REVIEWER_AND_SUGGESTION_COUNTS_ID
+            REVIEWER_AND_SUGGESTION_COUNTS_MODEL_ID
         )
 
         if reviewer_and_suggestion_counts_model is None:
-            cls(
-                id=REVIEWER_AND_SUGGESTION_COUNTS_ID,
-                translation_reviewer_counts_per_lang={},
-                translation_suggestion_counts_per_lang={},
+            reviewer_and_suggestion_counts_model = cls(
+                id=REVIEWER_AND_SUGGESTION_COUNTS_MODEL_ID,
+                translation_reviewer_counts_by_lang={},
+                translation_suggestion_counts_by_lang={},
                 question_reviewer_count=0,
                 question_suggestion_count=0
-            ).put()
+            )
+            reviewer_and_suggestion_counts_model.put()
+            return reviewer_and_suggestion_counts_model
 
         return super(
```

Comment:
Maybe have "else" clause here since only one branch or the other is executed

Reply:
I can add it :) I didn't put it because I thought it was implied.